department-of-veterans-affairs/abd-vro
# Move svc-lighthouse-api to its own repo

## What was the problem?

The titular service has not been used for over a year, cluttering the repo, our SecRel pipelines, and LHDI environments.

Associated tickets or Slack threads:
- #2697 

## How does this fix it?[^1]

* Moves the service into its own repo at https://github.com/department-of-veterans-affairs/abd-vro-svc-lighthouse-api

## How to test this PR
- Remove deployments in escalating environments


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
